README_DEVELOPERS.md

```diff
@@ -68,6 +73,26 @@ drone exec --matrix TARGET=test
 
 ### In a Cloud CI
 
-We provide configuration YAML files for [Travis CI](https://travis-ci.org/) ([.travis.yml](.travis.yml)) and [Shippable](https://app.shippable.com/) ([shippable.yml](shippable.yml)). Enable one of them for your forked repo (we recommend Travis). 
+We provide configuration YAML files for [Travis CI](https://travis-ci.org/) ([.travis.yml](.travis.yml)) and [Shippable](https://app.shippable.com/) ([shippable.yml](shippable.yml)). Enable one of them for your forked repo (we recommend Travis).
 
 Please make sure that each pull request passes all automatic checks before submitting it.
+
+## Commit Message Guidelines
+
+We have a few rules of formatting commit messages and pull request titles:
+
+- Use the imperative mood in the subject line
+- Capitalize the subject line
+- Do not end the subject line with a period
+- Limit the subject line to 50 characters
+- Separate the subject from the body with a blank line
+- Wrap the body at 72 characters
+- Use the body to explain what and why vs how
+- Write the ID of the issue/ticket at the end of the subject line
+
+Samples:
+
+- `Mark dxCommand as DOMComponent child (ID)`
```

Comment:
Can we show two different examples here for a GitHub issue fix and for a Support Center issue fix? The would look like (#XXXX) or (TXXXXX).

Reply:
Is it acceptable for us to use the GitHub`s keywords for closing issues in subjects of PR? 
For example (close/resolve/fix #XXXX) and (TXXXXX)?

Reply:
@Aden-git It depends on the intent. You may want to auto-close or only mention an issue.

Reply:
@AlekseyMartynov I want to say, that `Fix some issue (Fix #XXXX)` looks strange.

Reply:
Indeed, it does.
"Resolve #XXXX" looks better in this particular case.

Reply:
I agree with you, but it doesn't correspond [this rule](https://github.com/DevExpress/DevExtreme/pull/1498/files#diff-0e526e7e18a850bef2d7783404ace69dR91). Maybe, for now, we stop closing the GitHub issues from subjects? We can write the keyword in the body, for example.

Reply:
> but it doesn't correspond this rule

Bingo! That is exactly my concern regarding these rules.